app/darwin_ios.m

```diff
@@ -247,11 +247,38 @@ UIEdgeInsets getDevicePadding() {
     return UIEdgeInsetsZero;
 }
 
-void showKeyboard() {
+void showKeyboard(int keyboardType) {
     GoAppAppDelegate *appDelegate = (GoAppAppDelegate *)[[UIApplication sharedApplication] delegate];
     GoInputView *view = appDelegate.controller.inputView;
 
     dispatch_async(dispatch_get_main_queue(), ^{
+        switch (keyboardType)
+        {
+            case 0:
```

Comment:
Java has nicely defined constants to make the numbers more readable - can you do the same for this obj-c code as well?